tools/pybind11Common.cmake

```diff
@@ -18,11 +18,7 @@ Adds the following functions::
 
 #]======================================================]
 
-# CMake 3.10 has an include_guard command, but we can't use that yet
-# include_guard(global) (pre-CMake 3.10)
-if(TARGET pybind11::pybind11)
-  return()
-endif()
+include_guard(GLOBAL)
```

Comment:
This does not work well in config mode if one does find_package multiple times...